prow/cmd/tide/config.md

```diff
@@ -0,0 +1,158 @@
+# Configuring Tide
+
+Configuration of Tide is located under the [prow/config.yaml](/prow/config.yaml) file. All configuration for merge behavior and criteria belongs in the `tide` yaml struct, but it may be necessary to also configure presubmits for Tide to run against PRs (see ['Configuring Presubmit Jobs'](#configuring-presubmit-jobs) below).
+
+Tide is configured by populating the following YAML structure. Depending on your criteria, some of the fields may by omitted.
```

Comment:
In the grand future I would love for this doc to be served from `tide` itself, so that this could be generated from code

Reply:
Recommend using active voice wherever possible!